### PR TITLE
WIP - run tests in container

### PR DESCRIPTION
**Pull request by TomasTomecek**
Do not merge yet!

TODO:
 - [ ] a successful run in CI
 - [ ] polished UX

Signed-off-by: Tomas Tomecek <ttomecek@redhat.com>

### CODE REVIEW

**Comment by TomasTomecek** [comment]
currently blocked on this: https://github.com/fedora-modularity/meta-test-family/issues/16

**Review by phracek — APPROVED** [review]
Thanks for the Pull Request. This is definitely needed.

File: Makefile [hunk 1]
```diff
@@ -16,6 +18,12 @@ run:
 test:
 	cd $(VERSION)/$(TARGET) && make test VERSION=$(VERSION)
 
+test-in-container:
```
Comment (phracek) [inline Makefile]: Does it make sense to mention here `test-container`? It depends on it, right? At least for first run.

Comment (TomasTomecek) [reply Makefile]: Yes, that make sense.

**Comment by TomasTomecek** [comment]
@phracek thanks for trying to rebase my branch: but it looks like you forgot to actually "rebase" it: you should have do `git pull --rebase origin master` instead of `git pull [origin] [master]`. I'll clean it up.

**Comment by rpitonak** [comment]
Work moved to #38

**Pull request closed without merging**